api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -129,29 +144,78 @@ paths:
               - satisfied
               - not_satisfied
               - not_received
-        - name: source
+        - name: has_changed_children
           in: query
           description: >-
-            Filter Signals/meldingen according to their source, accepted values
-            are derived from the database. Source `online` are Signals/meldingen
-            that were created in the public website.
+            Filters all parent signals that have changes or no changes in children. Accepts true,
+            false or both. When both true and false are given only parent Signals are returned.
           required: false
           schema:
-            type: string
-        - name: 'type'
+            type: boolean
+            example:
+              true: true
+              false: false
+        - name: kind
           in: query
           description: >-
-            Filter signals according to the currently assigned type, accepts
-            (a combination of) the values `SIG` a "Signal/Melding", `REQ` a "Request/Aanvraag", `QUE` a "QuestionVraag", `COM` a "Complaint/Klacht" and `MAI` a "Maintenance/Groot onderhoud"
+            Filter Signals according to whether they are a Signal ("signal"), A parent Signal
+            ("parent_signal") and/or a child Signal ("child_signal"). An additional option can be
+            used to select all Signals that are NOT a parent Signal, this option is
+            "exclude_parent_signal". This filter accepts (a combination of) the options "signal",
+            "parent_signal", "exclude_parent_signal" or "child_signal". Parent/Child Signals can
```

Comment:
What if I set both of these: "parent_signal", "exclude_parent_signal". Is that even meaningful?

Reply:
Nope, but it is possible.